core-web/src/main/java/org/silverpeas/core/webapi/profile/UserProfileEntity.java

```diff
@@ -170,8 +170,8 @@ public String getLastName() {
 
   @Override
   @XmlElement
-  public String geteMail() {
-    return Encode.forHtml(this.user.geteMail());
+  public String getEmailAddress() {
```

Comment:
Wow! I am very surprised that there is no JavaScript implementation impacts. Are you sure?

Reply:
Yes I'm sure now that I have just fixed some missing js.